spinegeneric/cli/generate_figure.py

```diff
@@ -368,6 +374,88 @@ def compute_statistics(df):
     return df, stats
 
 
+def output_text(stats, metric):
+    """
+    # Write statistical results into text file
+    :param stats: dict with stat resutls
+    :param metric: currently processed metric (e.g, dti_fa,...)
+    """
+
+    def check_p_value(p_val):
+        """
+        Check if p-value is lower than 0.01, if so, change it to "<0.01", otherwise, round it to two decimals
+        :param p_val: input p-value
+        :return: p_val: processed p-value (replaced by "<0.01" or rounded to two decimals)
+        """
+        if p_val < 0.01:
+            p_val = "<0.01"
+        else:
+            p_val = round(p_val, 2)
+
+        return p_val
```

Comment:
it is usually cleaner to stick with a single output format. Here, it is either a string or a float.